docs/api.md

```diff
@@ -647,6 +628,29 @@ module.
 * See [`sonic-boom` ⇗](https://github.com/mcollina/sonic-boom)
 * See [Extreme mode ⇗](/docs/extreme.md)
 
+<a id="pino-final"></a>
+### `pino.final(logger, handler) => Function`
+
+The `pino.final` method supplies an exit listener function that can be 
+supplied to process exit events such as `exit`, `uncaughtException`, 
+`SIGHUP` and so on. 
+
+The exit listener function will call the supplied `handler` function 
+with a `finalLogger` instance. The `finalLogger` is a specialist 
+logger that synchronously flushes on every write. This is important 
+to gaurantee final log writes, both when using `pino.destination` or
+`pino.extreme` targets. 
+
+Since final log writes cannot be gauranteed with normal Node.js streams, 
+if the `destination` parameter of the `logger` supplied to `pino.final`
+is a Node.js stream `pino.final` will throw. It's highly recommended
+for both performance and safety to use `pino.destination`/`pino.extreme`
+destinations.
+
+* See [`destination` parameter](#destination)
+* See [Exit logging help](/docs/help.md#exit-logging)
+* See [Extreme mode ⇗](/docs/extreme.md)
```

Comment:
Can you please add an example here, or link to the "log loss prevention" section?